### ISSUE TITLE
TRILEGAL server busy

### ISSUE THREAD

**Issue by DJArmstrong** [timothydmorton/VESPA#22]
I've recently been using some of your TRILEGAL code to call the webserver, and noticed that it doesn't allow for the server being busy and rejecting the request (the code then gets stuck waiting for an output file that doesn't come).

To get round this I rebuilt the get_trilegal script in pure python, and added some elements to allow for the server being busy. Would you be interested in the code? If so I'll set up a pull request.

I'm not sure if this is best listed as an 'issue' but couldn't think where else to ask.

**Comment by timothydmorton** [comment]
Absolutely; that's been on a to-do list somewhere in my brain for awhile; this original perl script was provided to me years ago by Leo Girardi and not really touched since--- a python implementation would be very welcome.  PR away-- thanks!

**Comment by DJArmstrong** [comment]
No problem. Where do you want me to add it? In essence it's a function and a one-line call to it.

**Comment by timothydmorton** [comment]
How about you add your function to https://github.com/timothydmorton/VESPA/blob/master/vespa/stars/trilegal.py and then rewrite the contents of [get_trilegal](https://github.com/timothydmorton/VESPA/blob/176ffecdb3e4c93e73135e99f1ca03200221fbd1/vespa/stars/trilegal.py#L22) to call your function instead of `Popen`ing the perl script?
